packages/playwright-core/src/server/chromium/crPage.ts

```diff
@@ -333,6 +333,8 @@ export class CRPage implements PageDelegate {
   }
 
   async setInputFilePaths(progress: Progress, handle: dom.ElementHandle<HTMLInputElement>, files: string[]): Promise<void> {
+    if (this._browserContext._browser._interceptSetInputFilePaths)
```

Comment:
I'd rather revert the change that fixed lastModified, this becomes too hairy. Maybe we can move `convertInputFiles` logic to the server and let it properly handle the android case?